src/server/game/Grids/Notifiers/GridNotifiers.h

```diff
@@ -177,6 +177,42 @@ namespace Trinity
         }
     };
 
+    struct TC_GAME_API ChatMessageDistDeliverer
+    {
+        WorldObject const* i_source;
+        WorldPacket const* i_message;
+        uint32 i_phaseMask;
+        float i_distSq;
+        uint32 team;
+        Player const* skipped_receiver;
+        ChatMessageDistDeliverer(WorldObject const* src, WorldPacket const* msg, float dist, bool own_team_only = false, Player const* skipped = nullptr)
+            : i_source(src), i_message(msg), i_phaseMask(src->GetPhaseMask()), i_distSq(dist * dist)
+            , team(0)
+            , skipped_receiver(skipped)
+        {
+            if (own_team_only)
+                if (Player const* player = src->ToPlayer())
```

Comment:
Are you sure about `=` and not `==`? If you meant `=` then I'd suggest: `if (p* player = ToPlayer(); p != nullptr)`